tests/ci/ci_runner.py

```diff
@@ -102,7 +104,7 @@ def get_counters(fname):
         "PASSED": set([]),
         "FAILED": set([]),
         "SKIPPED": set([]),
-    }
+    }  # type: Dict[str, Any]
```

Comment:
`Set[str]`

Reply:
I put my best reasonable efforts to pass the style_check